src/state/user/hooks.tsx

```diff
@@ -349,14 +349,8 @@ export function useLiquidityPositionTokenPairs(): [Token, Token][] {
 
 export function useShowLiveChart(): boolean {
   const { chainId } = useActiveWeb3React()
-  let showLiveChart = useSelector((state: AppState) => state.user.showLiveCharts)
-  if (typeof showLiveChart?.[chainId] !== 'boolean') {
-    showLiveChart = defaultShowLiveCharts
-  }
-
-  const show = showLiveChart[chainId]
-
-  return !!show
+  const showLiveChart = useSelector((state: AppState) => state.user.showLiveChart)
+  return typeof showLiveChart !== 'boolean' ? defaultShowLiveCharts[chainId] : showLiveChart
```

Comment:
we no more need `defaultShowLiveCharts`, remove this also

Reply:
okay, updated